tests/integration/test_no_args.py

```
@@ -0,0 +1,43 @@
+    async def fake_main(paths: cabc.Iterable[Path], _concurrency: int) -> int:
+        captured.extend(paths)
+        return 0
+
+    monkeypatch.setattr(cli_module, "main", fake_main)
+    monkeypatch.chdir(tmp_path)
+    monkeypatch.setattr(sys, "argv", ["nixie"])
+
+    with pytest.raises(SystemExit) as excinfo:
+        cli_module.cli()
+
+    exc = typ.cast(SystemExit, excinfo.value)
```

Ruff (0.12.2)

Add quotes to type expression in `typing.cast()`

Add quotes

(TC006)